### PR TITLE
Driver image uploader

### PR DESCRIPTION
**Pull request by ArtyomKosakyan**
Added functionality for uploading taxi licence image. Added functionality for Admin for view licence images in drivers list.

### CODE REVIEW

**Review by iwazowsky — APPROVED** [review]
Looks good to me

File: src/components/RegistrationForm.js [hunk 1]
```diff
@@ -94,14 +94,14 @@ const RegistrationForm = ({
 };
 
 RegistrationForm.defaultProps = {
-  children: <></>,
+  children: [<></>],
```
Comment (vnersisyan-vineti) [inline src/components/RegistrationForm.js]: `children: null`

File: src/components/RegistrationForm.js [hunk 2]
```diff
@@ -94,14 +94,14 @@ const RegistrationForm = ({
...
   settings: false,
 };
 
 RegistrationForm.propTypes = {
   data: PropTypes.arrayOf(PropTypes.any).isRequired,
   header: PropTypes.string.isRequired,
-  children: PropTypes.element,
+  children: PropTypes.arrayOf(PropTypes.element),
```
Comment (vnersisyan-vineti) [inline src/components/RegistrationForm.js:104]: NIT: `children: PropTypes.shape({})`

File: src/helpers/SettingsHelper.js [hunk 3]
```diff
@@ -40,8 +40,18 @@ const SettingsHelper = ({
       ...fields,
       alert: '',
     };
-    parameter[event.target.id] = event.target.value;
-    setFields(parameter);
+    if (event.target.files && event.target.files[0]) {
```
Comment (vnersisyan-vineti) [inline src/helpers/SettingsHelper.js]: saw it 2nd time, move this to some helper

File: src/utils/ProfileInfoTable.js [hunk 4]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 
 const ProfileInfoTable = ({ fieldsData }) => {
   const fields = Object.entries(fieldsData).map((key) => (
```
Comment (vnersisyan-vineti) [inline src/utils/ProfileInfoTable.js]: try to destruct first element of key array in argument section

**Review by vnersisyan-vineti — CHANGES_REQUESTED** [review]
Some minor comments

File: src/helpers/SettingsHelper.js (hunk 3 above)
Comment (ArtyomKosakyan) [reply src/helpers/SettingsHelper.js]: Solved

File: src/components/RegistrationForm.js (hunk 2 above)
Comment (ArtyomKosakyan) [reply src/components/RegistrationForm.js:104]: But I have array, which contains jsx elements

File: src/components/RegistrationForm.js (hunk 1 above)
Comment (ArtyomKosakyan) [reply src/components/RegistrationForm.js]: solved

**Review by vnersisyan-vineti — APPROVED** [review]
LGTM

**Pull request merged**